config/application.rb

```diff
@@ -39,5 +39,9 @@ class Application < Rails::Application
     config.webpack.output_dir  = "public/assets/webpack"
     config.webpack.public_path = "assets/webpack"
     config.webpack.dev_server.enabled = false
+
+    # Loading Grape API framework.
+    config.paths.add File.join("app", "api"), glob: File.join("**", "*.rb")
+    config.autoload_paths += Dir[Rails.root.join("app", "api", "*")]
```

Comment:
Not really needed if you put it inside of `lib/api` for example. Otherwise, you should add it in the same way it's done above for the `lib` and `app/validators` directories...